win_configure/msi/wix/prep.bat

```diff
@@ -0,0 +1,257 @@
+@echo off
+REM This script will generate the componentgroup directory structure
+
+REM parsing command line arguments
+set argC=0
+for %%x in (%*) do Set /A argC+=1
+
+if %argC% LSS 1 (
+	echo 'Usage %0 "binaries_path=path_to_binaries_dir"'
+	exit /b 1
+) else (
+	set %1
+	@echo %binaries_path%
+)
+
+REM parsing the absolute path of this file to find PBS prefix directory
+SET PBS_prefix=''
+@setlocal enableextensions enabledelayedexpansion
+set variable=%~dp0
+if "x!variable:~-29!"=="xpbspro\win_configure\msi\wix\" (
+	set variable=!variable:~0,-29!
+) else (
+@echo "Failed to parse PBS prefix location"
```

Comment:
You already has 'echo off' so you don't need '@echo' or '@if' this is useful only when 'echo on'